ovirt-engine-metrics.spec.in

```diff
@@ -70,6 +70,10 @@ rm -rf %{buildroot}
 %doc README.md
 
 %changelog
+* Tue Nov 28 2023 Sandro Bonazzola <sandro.bonazzola@gmail.com> - 1.6.2-1
```

Comment:
Could you please add following change?
```
-  Fix comparison logic in ovirt_initial_validations Ansible role https://github.com/oVirt/ovirt-engine/issues/869
```

Reply:
It was the only important change since 1.6.1

Reply:
done